content/api/power-board.md

```diff
@@ -23,3 +23,30 @@ r.power_board.power_on()
 {{% notice warning %}}
 Because the motor board is powered through these power outputs, whilst the power is off, you won't be able to control your motors. This will register as a missing board and your code will break if you try and control them.
 {{% /notice %}}
+
+## Buzzing :bee:
+The power board has a piezo buzzer which can buzz.
+
+The `buzz` function accepts multiple parameters, depending on what you want to play. The first argument is the duration of the beep, in seconds. The later arguments are either the note you want to play, or the frequency of the buzzer (in Hz). You have to specify which of note or frequency you're passing using a keyword argument, your code will fail otherwise.
+
+Theoretically, the piezo buzzer will buzz at any provided frequency, however humans can only hear between [20Hz and 20000Hz](https://en.wikipedia.org/wiki/Hearing_range#Humans).
+
+{{% notice tip %}}
+Calling `buzz` is non-blocking. Meaning it doesn't actually wait for the piezo to stop buzzing before continuing with your code. If you want duration to wait for the buzzing to stop, add a `sleep` afterwards!
```

Comment:
We should clarify what a `sleep` is -- unless you happen to know about `time.sleep()` then just `sleep` doesn't mean much.